addon/components/datepicker-support.js

```diff
@@ -82,6 +82,12 @@ export default Ember.Mixin.create({
            .apply(element, Array.prototype.concat.call(['update'], dates));
   },
 
+  _updateLanguage: Ember.observer('language', function() {
```

Comment:
Could you please follow the naming style and call the method `didChangeLanguage` as it's an observer?